lib/tools/adb-commands.js

```diff
@@ -1626,6 +1626,7 @@ methods.runInImeContext = async function runInImeContext (ime, fn) {
   if (originalIme === ime) {
     log.debug(`The original IME is the same as '${ime}'. There is no need to reset it`);
   } else {
+    await this.enableIME(ime);
```

Comment:
does this return zero code if the given IME is already enabled?

Reply:
Yes
```
kazu$ adb shell ime enable io.appium.settings/.AppiumIME
Input method io.appium.settings/.AppiumIME: already enabled
kazu$ echo $?
0
```